api/schemas/flow-audio.json

```
@@ -33,15 +33,18 @@
           "properties": {
             "sample_rate": {
               "description": "The fixed number of samples per second.",
-              "type": "integer"
+              "type": "integer",
+              "exclusiveMinimum": 0
             },
             "channels": {
               "description": "The channel count.",
-              "type": "integer"
+              "type": "integer",
+              "exclusiveMinimum": 0
             },
             "bit_depth": {
-              "description": "The number of significant bits used to represent the audio sample. The minumum number of bytes then equals `round_up(bit_depth / 8)`.",
-              "type": "integer"
+              "description": "The number of significant bits used to represent the audio sample. The minumum number of bytes then equals `round_up(bit_depth / 8)`. If codec is `audio/x-raw-int` or `audio/x-raw-float`, bit_depth must be set.",
```

Slight improvement regarding float vs double:
```suggestion
              "description": "The number of significant bits used to represent the audio sample. The minumum number of bytes then equals `round_up(bit_depth / 8)`. If codec is `audio/x-raw-int` bit_depth must be set. If codec is `audio/x-raw-float` bit_depth must be set to 32 or 64",
```